### PR TITLE
V0.8.0

### PR DESCRIPTION
**Pull request by MegaByte875**

### CODE REVIEW

**Comment by kqzh** [comment]
do we need update README and other doc's nebula version to 2.5.0?

File: pkg/nebula/meta_client.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func (m *metaClient) GetSpace(spaceName string) (*meta.SpaceItem, error) {
 	if err != nil {
 		return nil, err
 	}
-	if resp.Code != meta.ErrorCode_SUCCEEDED {
+	if resp.Code != nebula.ErrorCode_E_DISCONNECTED {
```
Comment (veezhang) [inline pkg/nebula/meta_client.go:121]: `ErrorCode_E_DISCONNECTED` = > `ErrorCode_SUCCEEDED` ?

File: pkg/nebula/storage_client.go [hunk 2]
```diff
@@ -107,40 +94,24 @@ func (s *storageClient) TransLeader(spaceID nebula.GraphSpaceID, partID nebula.P
...
-				log.Error(err, "TransLeader failed")
-				return err
-			}
-			log.Info("TransLeader returned")
-			if len(resp.Result_.FailedParts) > 0 {
-				return errors.Errorf("Trans leader failed")
-			}
 			return nil
-		} else if resp.Result_.FailedParts[0].Code == storage.ErrorCode_E_LEADER_CHANGED {
-			return errors.Errorf("leader changed, please request to leader")
```
Comment (veezhang) [inline pkg/nebula/storage_client.go:143]: Why remove the logic about `ErrorCode_E_LEADER_CHANGED` ?

File: pkg/nebula/meta_client.go (hunk 1 above)
Comment (MegaByte875) [reply pkg/nebula/meta_client.go]: Fixed

File: pkg/nebula/storage_client.go (hunk 2 above)
Comment (MegaByte875) [reply pkg/nebula/storage_client.go:143]: These codes are unused

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**